src/System.Management.Automation/engine/remoting/common/RunspaceConnectionInfo.cs

```diff
@@ -3260,6 +3280,19 @@ private void CreateContainerProcessInternal()
             ErrorCode = error;
         }
 
+        /// <summary>
+        /// Get Command to launch container process based on instance properties.
+        /// </summary>
+        private string GetContainerProcessCommand()
+        {
+            return string.Format(System.Globalization.CultureInfo.InvariantCulture,
+                        @"{{""CommandLine"": ""{0} {1} -NoLogo {2}"",""RestrictedToken"": {3}}}",
+                        Executable, 
```

Comment:
It feels cleaner if the Executable loop variable is passed in as a parameter.  I know we need the property set for error handling, but this code is a bit confusing without realizing Executable is a loop variable.